test/utils/net.go

```diff
@@ -65,14 +63,8 @@ func PlatformPorts(includePublicPorts bool) (ports []string) {
 		ServicePort("core-command"),
 		ServicePort("vault"),
 		ServicePort("consul"),
-		ServicePort("kong-database"),
 		ServicePort("redis"),
 	)
-	if includePublicPorts {
-		ports = append(ports,
-			ServicePort("kong"),
-		)
-	}
```

Comment:
It would be 8000 and 8443.  And add proxy-auth microservice as well.  I have a PR for adding the checks back in.  https://github.com/canonical/edgex-snap-testing/pull/164

Postgres (5432) is actually the offending service that is being removed.

Reply:
Got it, thanks.
So we can merge this now and then work on #164 once https://github.com/edgexfoundry/edgex-go/pull/4244 is merged.

Reply:
👌